### PR TITLE
UPDATE documentation

### PR DESCRIPTION
**Pull request by Lodimup**
UPDATE incorrect import 
ADD callback uri information

## Checklist 🧢

- [x] Documentation
- [ ] Tests
- [ ] Ready to be merged

### CODE REVIEW

File: docs/docs/providers/42.md [hunk 1]
```diff
@@ -13,7 +13,9 @@ https://api.intra.42.fr/apidoc/guides/web_application_flow
 
 ## Configuration
 
-https://profile.intra.42.fr/oauth/applications/new
+https://profile.intra.42.fr/oauth/applications/new  
+The callback uri has to be  
+DOMAIN.TLD/api/auth/callback/42-school
```
Comment (lluia) [inline docs/docs/providers/42.md:18]: We should highlight this better in the **Getting Started** docs instead 🤔 
https://next-auth.js.org/getting-started/rest-api#getpost-apiauthcallbackprovider

cc. @balazsorban44

File: docs/docs/providers/42.md [hunk 2]
```diff
@@ -26,10 +28,10 @@ You can override any of the options to suit your own use case.
 ## Example
 
 ```js
-import FortyTwoProvider from "next-auth/providers/42";
+import FortyTwo from "next-auth/providers/42-school";
```
Comment (lluia) [inline docs/docs/providers/42.md:31]: Let's keep consistency with documentation on other providers.
```suggestion
import FortyTwoProvider from "next-auth/providers/42-school";
```

File: docs/docs/providers/42.md [hunk 3]
```diff
@@ -26,10 +28,10 @@ You can override any of the options to suit your own use case.
...
 ...
 providers: [
-  FortyTwoProvider({
+  FortyTwo({
```
Comment (lluia) [inline docs/docs/providers/42.md:34]:
```suggestion
  FortyTwoProvider({
```

**Review by 0ubbe — CHANGES_REQUESTED** [review]

File: docs/docs/providers/42.md (hunk 1 above)
Comment (lluia) [reply docs/docs/providers/42.md:18]: FYI https://github.com/nextauthjs/next-auth/pull/4065

Comment (lluia) [reply docs/docs/providers/42.md:18]: Let's remove this part as well; I opened a PR to explain this part better in the **Getting Started** tutorial.

**Review by 0ubbe — CHANGES_REQUESTED** [review]

File: docs/docs/providers/42.md (hunk 1 above)
Comment (lluia) [inline docs/docs/providers/42.md:18]:
```suggestion
```
We need to remove this as it's already going to be explained here: https://github.com/nextauthjs/next-auth/pull/4065

**Review by 0ubbe — CHANGES_REQUESTED** [review]

File: docs/docs/providers/42.md (hunk 1 above)
Comment (Lodimup) [reply docs/docs/providers/42.md]: Thanks for the hard work!

File: docs/docs/providers/42.md [hunk 4]
```diff
@@ -13,7 +13,7 @@ https://api.intra.42.fr/apidoc/guides/web_application_flow
 
 ## Configuration
 
-https://profile.intra.42.fr/oauth/applications/new
+https://profile.intra.42.fr/oauth/applications/new
```
Comment (balazsorban44) [inline docs/docs/providers/42.md:16]:
```suggestion
https://profile.intra.42.fr/oauth/applications/new
```

**Review by balazsorban44 — APPROVED** [review]

**Pull request merged**